website/blog/2020-11-04-2.2.0.md

```diff
@@ -0,0 +1,865 @@
+}
+
+type T = (this: boolean, a: number) => boolean;
+
+// Prettier 2.1
+function f(this: string, a: number) {}
+
+type T = (a: number) => boolean;
+
+// Prettier 2.2
+function f(this: string, a: number) {
+}
+
+type T = (this: boolean, a: number) => boolean;
+```
+
+#### Support `BigIntLiteralTypeAnnotation` and `BigIntTypeAnnotation` ([#9523](https://github.com/prettier/prettier/pull/9523) by [@fisker](https://github.com/fisker))
+
+Add support for `BigIntLiteralTypeAnnotation` and `BigIntTypeAnnotation` in Flow.
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+const foo: bigint = 1n;
+const bar: baz<1n> = 1n;
+
+// Prettier 2.1
+Error: unknown type: "BigIntTypeAnnotation"
+    at ...
+
+// Prettier 2.2
+const foo: bigint = 1n;
+const bar: baz<1n> = 1n;
+```
+
+#### Treat more types as "simple" type ([#9543](https://github.com/prettier/prettier/pull/9543) by [@fisker](https://github.com/fisker))
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+const foo1: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<symbol> = a
+const foo2: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<"STRING"> = a;
+const foo3: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<0> = a;
+
+// Prettier 2.2
+const foo1: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<
+  symbol
+> = a;
+const foo2: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<
+  "STRING"
+> = a;
+const foo3: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<
+  0
+> = a;
+
+// Prettier 2.1 (typescript and flow parser)
+const foo1: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<symbol> = a
+const foo2: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<"STRING"> = a;
+const foo3: Fooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo<0> = a;
+```
+
+#### Fix missing parentheses around `prettier-ignore`d type assertions ([#9553](https://github.com/prettier/prettier/pull/9553) by [@fisker](https://github.com/fisker))
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+transform(
+  // prettier-ignore
+  (pointTransformer: (Point => Point))
+);
+
+// Prettier 2.1
+transform(
+  // prettier-ignore
+  pointTransformer: (Point => Point)
+);
+
+// Prettier 2.2
+transform(
+  // prettier-ignore
+  (pointTransformer: (Point => Point))
+);
+```
+
+#### Improve comment types detection ([#9563](https://github.com/prettier/prettier/pull/9563) by [@fisker](https://github.com/fisker))
+
+<!-- prettier-ignore -->
+```jsx
+// Input
+foo/*::<bar>*/(baz);
+class Foo {
+  bar( data: Array<string>) {}
+}
+
+// Prettier 2.2
+foo/*:: <bar> */(baz);
+class Foo {
+  bar(data: Array/*:: <string> */) {}
+}
+
+// Prettier 2.1
+foo/*:: <bar> */(baz);
+class Foo {
+  bar(data: Array<string>) {}
+}
+```
+
+### CSS
+
+#### Improve custom properties format ([#9209](https://github.com/prettier/prettier/pull/9209) by [@fisker](https://github.com/fisker))
```

Comment:
So, just changelog didn't get reverted?

Reply:
Oh, this is my mistake. I'll fix.

Reply:
https://github.com/prettier/prettier/pull/9589/commits/2527e8041ce32d94eb14344cb9d71b3031ca1102#diff-b3b191bef4c95bbbb93aeae7a119d6cf4e39e6d1710e4a956dd71b46503189abL836